ios/Classes/BetterPlayer.m

```diff
@@ -238,6 +238,15 @@ - (void)setDataSourcePlayerItem:(AVPlayerItem*)item withKey:(NSString*)key{
     _playerRate = 1;
     [_player replaceCurrentItemWithPlayerItem:item];
 
+    CMTime durationIn5Minutes = CMTimeMakeWithSeconds(300, 1);
+    const BOOL isLive = CMTIME_IS_INDEFINITE(item.duration);
```

Comment:
`isLive` condition is also
If this is really needed, can keep it. 
But if this is kind of additional code, better to remove it as of now.